app/Http/Controllers/MinesController.php

```
@@ -0,0 +1,273 @@
+                'tile' => $data['tile'],
+                'payout_multiplier' => $currentMultiplier
+            ];
+            $mine->tiles_revealed = $tilesRevealed;
+        }
+
+        $mine->save();
+
+        return new MineResource($mine);
+    }
+
+    public function cashout(Request $request)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)